scripts/build-tools.sh

```diff
@@ -42,11 +42,10 @@ build_fuzzer()
 
 main()
 {
```

Comment:
I don't really support referencing PR issues from git commit log, as most of the people will work decoupled from the github. 

I prefer for future patches to add a brief description of the issue if it is possible and use referencing PRs as less as possible.

Reply:
> I don't really support referencing PR issues from git commit log, 
 
I'm not a fan either. In fact it's even worse because github [tends to lose code review data](https://github.com/zephyrproject-rtos/zephyr/pull/14444#issuecomment-575320941). However #2543 is a bug, not a PR

> I prefer for future patches to add a brief description of the issue if it is possible 

I added a reference to the alsa-utils commit, see commit message v2 (which github can't show the changes of, demonstrating my earlier point).

As expected, this reference to the alsa-utils commit trigger this new and only checkpatch warning: `WARNING: Unknown commit id 'caf77a93cef5`

> as most of the people will work decoupled from the github.

I don't see how that's possible, I mean not without losing a lot. 

> and use referencing PRs as less as possible.

I totally agree we should still try as much as possible! Thanks for the reminder.